src/setupTests.js

```
@@ -20,15 +20,19 @@ vi.mock('../supabaseClient', () => ({
-    })),
+    from: vi.fn(() => {
+      const chainable = {};
+      chainable.select = vi.fn(() => chainable);
+      chainable.insert = vi.fn(() => chainable);
+      chainable.update = vi.fn(() => chainable);
+      chainable.delete = vi.fn(() => chainable);
+      chainable.eq = vi.fn(() => chainable);
+      chainable.is = vi.fn(() => chainable);
+      chainable.single = vi.fn().mockResolvedValue({ data: null, error: null });
+      chainable.order = vi.fn().mockResolvedValue({ data: [], error: null });
+      chainable.then = vi.fn((resolve) => resolve({ data: [], error: null }));
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)